packages/hippodrome-ui/src/App.tsx

```diff
@@ -13,7 +13,7 @@ export const App = () => (
       <Router>
         <Layout>
           <Switch>
-            <Route path="/transaction/mint/:asset/:to/:nonce" exact>
+            <Route path="/transaction/mint/:asset/:to/:nonce/:outputToken" exact>
```

Comment:
I see chances of this getting removed if user closes the tab. What I suggest is saving the token in local storage as well and then clear when a swap is complete

Reply:
yeah doing both is good... but let's save that until after the hackathon

Reply:
what I think we should do post hackathon is save state to the blockchain itself using kasumah-logger

Reply:
yea make sense